examples/vision_gpt_explain_image/app.py

```diff
@@ -0,0 +1,50 @@
+import agenta as ag
+from openai import OpenAI
+
+
+client = OpenAI()
+
+
+SYSTEM_PROMPT = "You are an expert in reading images you look into details, you answer in accurate language."
+HUMAN_PROMPT = "Please compare two images"
+
+ag.init(app_name="explain_image", base_name="app")
```

Comment:
why do we need the params here? why not keep it empty?

Reply:
No reason, actually. I'll keep it empty.